packages/admin/admin/src/stack/useIsLastVisibleSwitch.ts

```diff
@@ -0,0 +1,15 @@
+import { useStackApi } from "./Api";
+import { useStackSwitchApi } from "./Switch";
+
+export const useIsLastVisibleSwitch = () => {
```

Comment:
Naming: Wouldn't useIsActiveStackSwitch be better?

Reply:
Yes 😄